qa/rpc-tests/bip65-cltv-p2p.py

```diff
@@ -43,7 +43,7 @@ def __init__(self):
     def setup_network(self):
         # Must set the blockversion for this test
         self.nodes = start_nodes(self.num_nodes, self.options.tmpdir,
-                                 extra_args=[['-debug', '-whitelist=127.0.0.1', '-blockversion=3']],
+                                 extra_args=[['-debug', '-whitelist=127.0.0.1', '-blockversion=3', '-buriedsfparams=bip65:1351']],
```

Comment:
Nit: Would it make sense to revert https://github.com/bitcoin/bitcoin/pull/8391/files#diff-30641a8ce712805f484b8305049f04b3 to the old code as well?